test/acceptance/tests/basic/terraform/basic-install/main.tf

```diff
@@ -299,6 +323,14 @@ resource "aws_iam_role" "task" {
       },
     ]
   })
+
+  // TODO: When using the auth method, mesh-task should add these tags if the task role
+  // is passed in.
```

Comment:
It doesn't look like we can add individual tags to an existing role that is passed in.

Terraform does support [managing individual tags for certain resources](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/guides/resource-tagging#managing-individual-resource-tags), but there is no resource for IAM role tags, unfortunately.

For now, if a user passes their own role, they will need to add these tags to the role on their side (if they have ACLs enabled and are using the auth method).